lib/graphiti/active_graph/adapters/active_graph.rb

```diff
@@ -170,6 +168,16 @@ def resource_association_value(rel_map)
           rel_map[:object]
         end
       end
+
+      def find_record(x)
+        if Graphiti.config.respond_to?(:allow_sidepost) && !Graphiti.config.allow_sidepost
```

Comment:
Once we have `attr_accessor :allow_sidepost` here in this gem, will we need this check `Graphiti.config.respond_to?(:allow_sidepost)`? I think we will be able to access the config without checking with `respond_to`

Reply:
makes sense